modules/bitgo/example/ts/avaxp/multisig-crosschain-export-import.ts

```diff
@@ -0,0 +1,252 @@
+import { Avalanche, BinTools, BN, Buffer } from 'avalanche';
```

Comment:
Can we get a detailed description outlining this example at the top of this like the other examples?

Reply:
okay, done

Reply:
Thanks!